chore: archiving design system sketch plugin (#9)
salesforce-ux/design-system-sketch: README.md

> # Lightning Design System Plugin for Sketch [No Longer Supported]

Reviewer: ```suggestion
#  [End-of-Support] Lightning Design System Plugin for Sketch ```

> > As of 246, The Lightning Design System Plugin for Sketch is no longer supported & actively maintained by the contributors.

Reviewer: ```suggestion
## 📣 Disclaimer

As of Summer `23 (SLDS 2.21.0), we're officially announcing the end-of-support (EOS) for the Lightning Design System Plugin for Sketch plugin. If you're wondering what that means, simply put, we won't be providing any further updates, bug fixes, or security patches for this plugin. 

We understand that many of you have come to rely on this plugin, and we sincerely appreciate your support. But just like all good things, this too must come to an end. 

This doesn't mean that the plugin will stop working immediately. You're free to continue using it, but do so at your own risk. Since we're not providing security updates anymore, it's possible that future vulnerabilities might go unpatched.

We're archiving this repository, but the code will always be here. Feel free to fork it, tweak it, and make it your own. 

Thanks for understanding and for your continued support.
```
